recipes-support/librealsense/librealsense_1.10.0.bb

```diff
@@ -88,3 +88,10 @@ FILES_${PN}-tests += "\
 
 # Configuration is not required
 do_configure[noexec] = "1"
+
+#Skip ldflags QA check
```

Comment:
The problem is that the build system LDFLAGS are not passed to make. 

Proper fix is to ```LDFLAGS=${LDFLAGS}```  above.